src/org/zaproxy/zap/extension/callback/ui/CallbackModel.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2017 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.zaproxy.zap.extension.callback.ui;
+
+import org.parosproxy.paros.network.HttpMessage;
+
+import java.util.Date;
+
+public class CallbackModel {
+
+    private String handler;
+    private Date date;
+    private HttpMessage httpMessage;
```

Comment:
Maybe it should use a `HistoryReference` (keeping the `HttpMessage`s in memory (longer that needed) tends to OOME, although in this case is not that problematic, since it has no response which are usually bigger).